compilers/concrete-compiler/compiler/lib/Bindings/Python/CompilerAPIModule.cpp

```diff
@@ -1014,6 +1015,7 @@ void mlir::concretelang::python::populateCompilerAPISubmodule(
           [](::concretelang::clientlib::ClientParameters clientParameters,
              ::concretelang::clientlib::KeySet &keySet,
              std::vector<lambdaArgument> args, const std::string &circuitName) {
```

Comment:
If I'm not wrong that should not be used anymore, this code need a cleanup. We now encrypt with exporter/decrypter

Reply:
So if you want to release gil for encryption let do that in export_scalar/export_tensor and decrypt

Reply:
Ok, removed.

Reply:
There are still in doc and tests.